examples/webgl_materials_envmaps_hdr.html

```diff
@@ -215,12 +219,53 @@
 
 				window.addEventListener( 'resize', onWindowResize, false );
 
+				function updateEnvMap() {
+
+					var newEnvMap = standardMaterial.envMap;
+
+					switch( params.envMap ) {
+
+						case 'None': newEnvMap = null; break;
+						case 'LDR': newEnvMap = ldrCubeRenderTarget.texture; break;
+						case 'HDR': newEnvMap = hdrCubeRenderTarget.texture; break;
+						case 'RGBM16': newEnvMap = rgbmCubeRenderTarget.texture; break;
+
+					}
```

Comment:
I messed up here: it's still necessary to check if the `*CubeRenderTarget` variables are `undefined`. Otherwise, selecting a different envMap in the GUI while textures are still loading would result in an error.